packages/core/src/Snippet/Actions.js

```diff
@@ -22,10 +22,10 @@ const useStyles = makeStyles({
   }
 });
 
-function SnippetActions({ onShare, onEmbed }) {
-  const classes = useStyles();
+function SnippetActions({ onShare, onEmbed, ...props }) {
+  const classes = useStyles(props);
   return (
-    <Grid className={classes.actionsContainer} container spacing={0}>
+    <Grid className={classes.root} container spacing={0}>
```

Comment:
I try avoid root since prop spreading can cause issues.

Reply:
I hear you but using `root` makes so much sense that I'm really debating whether we should:
 1. Take the time to actually check what classes does a child component contains and only pass those, similar to: https://github.com/CodeForAfrica/HURUmap-UI/blob/d965d7c1bb914a92f024d9b57099bbc5001aab00/packages/core/src/Snippet/index.js#L126-L138, or
 2.  Switch to back to `withStyles` HoC

Reply:
We can get back to this another time